client/src/components/Input/Input.tsx

```diff
@@ -15,56 +15,47 @@ import { Flex } from 'styles/global';
 import * as S from './Input.styles';
 
 const INPUT = {
-  input: ({ elementConfig, theme, handleClearValue, clearEnabled, ...otherProps }: InputProps) => (
+  input: ({ inputConfig, theme, handleClearValue, clearEnabled, ...otherProps }: InputProps) => (
     <S.Input
       themeType={theme}
       onBlur={e => {
         if (clearEnabled && handleClearValue) handleClearValue(e);
       }}
-      {...elementConfig}
+      {...inputConfig}
       {...otherProps}
     />
   ),
-  textarea: ({ elementConfig, theme, ...otherProps }: InputProps) => (
-    <S.Textarea themeType={theme} {...elementConfig} {...otherProps} />
+  textarea: ({ inputConfig, theme, ...otherProps }: InputProps) => (
+    <S.Textarea themeType={theme} {...inputConfig} {...otherProps} />
   ),
-  radio: ({ id, elementConfig, theme, ...otherProps }: InputProps) => (
+  radio: ({ id, inputConfig, theme, ...otherProps }: InputProps) => (
     <S.InputGroup>
       <Flex justifyFlexStart>
-        {elementConfig.options &&
-          Object.keys(elementConfig.options).map((option, i) => {
-            const values = elementConfig.options && Object.values(elementConfig.options);
-            return (
-              <S.RadioWrapper key={option} themeType={theme}>
-                <S.Radio id={option} name={id} {...elementConfig} {...otherProps} />
-                <label htmlFor={option}>{values && values[i]}</label>
-              </S.RadioWrapper>
-            );
-          })}
+        {inputConfig.options &&
+          Object.keys(inputConfig.options).map(option => (
+            <S.RadioWrapper key={option} themeType={theme}>
+              <S.Radio id={option} name={id} {...inputConfig} {...otherProps} />
+              <label htmlFor={option}>{inputConfig.options![option]}</label>
+            </S.RadioWrapper>
+          ))}
       </Flex>
     </S.InputGroup>
   ),
-  rating: ({ id, elementConfig, theme, ...otherProps }: InputProps) => (
-    <StarRating
-      id={id}
-      numOfStars={elementConfig.options && Object.values(elementConfig.options).length}
-      theme={theme}
-      elementConfig={elementConfig}
-      {...otherProps}
-    />
+  rating: ({ id, inputConfig, theme, ...otherProps }: InputProps) => (
+    <StarRating id={id} theme={theme} inputConfig={inputConfig} {...otherProps} />
```

Comment:
looks like now you can pass all props with spread operator, not `otherProps` only